### PR TITLE
Fix lint error for test cases in apptools

### PR DESCRIPTION
**Pull request by haoyunfeix**
@haoxli @Honry ,PTAL

### CODE REVIEW

**Comment by Honry** [comment]
:( CI check fails.

File: .travis.yml [hunk 1]
```diff
@@ -9,5 +9,7 @@ before_script:
   - sudo apt-get install python-demjson
   - sudo apt-get install libxml2-utils
   - pip install html5lib
+  - easy_install demjson
 
-script: python ./tools/lint/lint.py -p
+#script: python ./tools/lint/lint.py -p
+script: jsonlint -v apptools/apptools-manifest-tests/tcs_android/manifest32-negative/manifest.json
```
Comment (Honry) [inline .travis.yml]: Please remove this debug code.

**Comment by haoyunfeix** [comment]
@Honry PTAL

File: apptools/apptools-manifest-tests/tcs_android/manifest32-negative/manifest.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
 {
 "xwalk_package_id" : "org.xwalk.tests",
-"icons" : [{"src": "icon/icon.png","sizes": "128x128"}],
-}
\ No newline at end of file
+"icons" : [{"src": "icon/icon.png","sizes": "128x128"}]
```
Comment (Honry) [inline apptools/apptools-manifest-tests/tcs_android/manifest32-negative/manifest.json]: Is this accepted invalid json file?

**Comment by Honry** [comment]
LGTM

**Pull request merged**